readme.md

```diff
@@ -23,10 +23,30 @@ console.log(random(), random(), random());
 
 ## API
 
-### uniqueRandom(minimum, maximum)
+### uniqueRandom(minimum, maximum, options?)
 
 Returns a function, that when called, will return a random number that is never the same as the previous.
 
+#### minimum
+
+Type: `number`\
+The minimum number that can be returned.
+
+#### maximum
+
+Type: `number`\
+The maximum number that can be returned.
+
+#### options
+
+Type: `object`
+
+##### exhaustive
+
+Type: `boolean`\
+Default: `false`\
+Prevent extracted numbers regeneration before all possibilities are exhausted.
```

Comment:
This is not in sync with the TS docs.

Reply:
Just added the example and updates made to TS docs, too.